Dockerfile.arm

```diff
@@ -27,10 +27,28 @@ RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin"
 
 FROM multiarch/qemu-user-static:x86_64-arm as qemu
 FROM arm32v7/debian:buster-slim
+
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
```

Comment:
You added `ARG DEBIAN_FRONTEND="noninteractive"` definition in the `Dockerfile.arm64` image, but I think you forgot to add here it too!

Reply:
Made them match between the images. Looking at them all, I need to clean up the Dockerfiles. Maybe remove curl and just use git to pull the web browser since that's a package requirement already now.

Reply:
why remove this variable? I think `apt` should know it's running in non-interactive mode...